.github/workflows/status-checks.yaml

```diff
@@ -17,6 +17,8 @@ jobs:
           cache: npm
 
       - run: npm clean-install
+        env:
+          NPM_AUTH: ${{ secrets.NPM_AUTH }}
 
```

Comment:
You can use a [top-level `env` key](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#env) to make environment variables accessible to all jobs in a workflow or a [job-level `env` key](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idenv) for all steps in a job.

Either of these would prevent the need for you to duplicate the definition of the environment variable.